win32ss/user/user32/misc/exticon.c

```diff
@@ -469,17 +469,20 @@ static UINT ICO_ExtractIconExW(
 
                     icon = CreateIconFromResourceEx(imageData, entry->icHeader.biSizeImage, sig == 1, 0x00030000, cx[index], cy[index], flags);
 
+                    HeapFree(GetProcessHeap(), 0, cursorData);
```

Comment:
Not checking whether cursorData / imageData could be null?

Reply:
HeapFree(NULL) is legal (despite what MSDN says :p)